docs/sdks/client-sdks/javascript/Initialization.mdx

```diff
@@ -5,16 +5,13 @@ sidebar_position: 3
 
 import ApiOptionRef from '@site/src/components/ApiOptionRef';
 
-
 The Eppo JavaScript SDK is easy to initialize while offering robust customization options, making it adaptable to various use cases such as offline mode, custom caching requirements, and ultra-low-latency initialization.
 
 ## Initialize the SDK
 
-To complete basic initialization, you only need to provide an SDK key which you can created in the Eppo UI under **Configuration** > **Environments** > **API Keys**. 
+To complete basic initialization, you only need to provide an SDK key. [Create an SDK key](/sdks/sdk-keys) if you don't already have one. 
 
 ```javascript
-import * as EppoSdk from "@eppo/js-client-sdk";
```

Comment:
We need this same change in Quickstart.

Also, unfortunately*, our TypeScript types require `assignmentLogger` to be defined:

```ts
import { init } from "@eppo/js-client-sdk";

// One-time initialization to put in application bootup
await init({
  apiKey: "SDK_KEY",
  assignmentLogger: { logAssignment: (assignmentEvent)=> console.log('Send to warehouse: ', assignmentEvent) },
});
```

*this is probably something we should consider changing for dev-friendliess